resources/library-phpcs-ruleset.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<!-- PHP_CodeSniffer standard for OpenEuropa PHP libraries. -->
+<!-- See http://pear.php.net/manual/en/package.php.php-codesniffer.annotated-ruleset.php -->
+<ruleset name="OpenEuropa PHP library">
+    <description>Coding standard for OpenEuropa PHP libraries.</description>
+
+    <rule ref="./vendor/squizlabs/php_codesniffer/src/Standards/PSR2" />
+
+    <!-- Comments and multi-line statements should be indented with 4 spaces. -->
+    <rule ref="Generic.WhiteSpace.ScopeIndent">
+        <properties>
+            <property name="ignoreIndentationTokens" type="array" value="" />
+            <property name="exact" value="true" />
+        </properties>
+    </rule>
+
+    <!-- Arrays that span multiple lines should be indented with 4 spaces. -->
+    <rule ref="Generic.Arrays.ArrayIndent" />
+
+    <!-- Chained method calls should be indented with 4 spaces. -->
+    <rule ref="PEAR.WhiteSpace.ObjectOperatorIndent" />
+
+    <!-- The concatenation operator should be surrounded by spaces. -->
+    <rule ref="PSR12.Operators.OperatorSpacing" />
```

Comment:
I would add only bits of another PSR standard, let's stick with PSR2 for the moment, we can switch to PSR12 later on and, when doing that, we should release a new major version of the component as it will greatly impact people's code by creating very large diffs.

I've also discussed array and object indentation rules with @drupol, he explained me that PSR2 is not opinionated at all on that, so they are good to go for me!

Reply:
Yep, I just pushed a commit to remove the line.